### DISCUSSION TITLE
Bevy Scenes Rework?

### DISCUSSION THREAD

**Discussion by abocado18** [bevyengine/bevy#14905] [Q&A]
I have found this link https://github.com/bevyengine/bevy/discussions/9538. From what I understand, scenes in Bevy will get a complete rework. Should I wait until this rework is implemented or is there a point in learning the current implementation?

**Comment by jirisvd** [comment]
Just a heads-up, there's a newer updated proposal for the Scene/UI rework in [#14437](https://github.com/bevyengine/bevy/discussions/14437). It is currently being intensively worked on, see the [Discord working group thread](https://discord.com/channels/691052431525675048/1235758970703188008/threads/1264881140007702558) if you're interested.

As far as using the current scene system, I guess it depends whether you really need something to use right now or whether you're fine just using plain commands to spawn entities for the time being. Here's cart's comment replying to a question about when the new scene system will be ready: [link](https://discord.com/channels/691052431525675048/1264881140007702558/1275884248045191270). It's not guaranteed to land in 0.15 but at least important parts of it could be ready by that point.
